jslib/fetchDataStream.js

```diff
@@ -39,7 +39,7 @@ async function processLines(data, rowcb, endstreamcb) {
         }
     }
     return {remainder: data.slice(rowStart), eos: false};
-};
+}
```

Comment:
The semicolons are detected by eslint on VSCode but no error are raised for this file when running `$ npm run lint:javascript`. 
I guess it is a `js` file and not a `jsx`?